andino_firmware/src/hw.h

```diff
@@ -39,9 +39,9 @@ struct Hw {
   static constexpr int kLeftEncoderChannelBGpioPin{3};
 
   /// @brief Right encoder channel A pin. Connected to PC4 (digital pin 18, analog pin A4).
-  static constexpr int kRightEncoderChannelAGpioPin{18};
+  static constexpr int kRightEncoderChannelAGpioPin{14};
   /// @brief Right encoder channel B pin. Connected to PC5 (digital pin 19, analog pin A5).
-  static constexpr int kRightEncoderChannelBGpioPin{19};
+  static constexpr int kRightEncoderChannelBGpioPin{15};
```

Comment:
Although they will be unused, let's please create to constants for the IMU pins as well so this can help as a reference for the assembly process. Let's please be explicit about this purpose on the corresponding pin docstrings.